src/main/java/org/openrewrite/java/testing/cleanup/AssertEqualsBooleanToAssertBoolean.java

```diff
@@ -0,0 +1,109 @@
+
+    @Override
+    public String getDisplayName() {
+        return "Replace JUnit `assertEquals(false, <boolean>)` to `assertFalse(<boolean>)` / `assertTrue(<boolean>)`";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Using `assertFalse` or `assertTrue` is simpler and more clear.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return new JavaVisitor<ExecutionContext>() {
+
+            JavaParser.Builder<?, ?> javaParser = null;
+
+            private JavaParser.Builder<?, ?> javaParser(ExecutionContext ctx) {
+                if (javaParser == null) {
+                    javaParser = JavaParser.fromJavaVersion()
+                            .classpathFromResources(ctx, "junit-jupiter-api-5.9");
+                }
+                return javaParser;
+            }
+
+            @Override
+            public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx) {
+                J.MethodInvocation mi = (J.MethodInvocation) super.visitMethodInvocation(method, ctx);
+                if (ASSERT_EQUALS.matches(mi) && isBooleanLiteral(mi)) {
+                    StringBuilder sb = new StringBuilder();
+                    String assertMethod = Boolean.parseBoolean(((J.Literal) mi.getArguments().get(0)).getValueSource())
+                            ? "assertTrue" : "assertFalse";
+                    Statement assertion = (Statement) mi.getArguments().get(1);
+                    if (mi.getSelect() == null) {
+                        maybeRemoveImport("org.junit.jupiter.api.Assertions");
+                        maybeAddImport("org.junit.jupiter.api.Assertions", assertMethod);
+                    } else {
+                        sb.append("Assertions.");
+                    }
+                    sb.append("#{}(#{any(java.lang.Boolean)}");
+                    Object[] args;
+                    if (mi.getArguments().size() == 3) {
+                        args = new Object[]{assertMethod, assertion, mi.getArguments().get(2)};
+                        sb.append(", #{any()}");
+                    } else {
+                        args = new Object[]{assertMethod, mi.getArguments().get(1)};
+                    }
+                    sb.append(")");
+                    JavaTemplate t;
+                    if (mi.getSelect() == null) {
+                        t = JavaTemplate.builder(sb.toString())
+                                .contextSensitive()
+                                .staticImports(String.format("org.junit.jupiter.api.Assertions.%s", assertMethod))
+                                .javaParser(javaParser(ctx))
+                                .build();
+                    } else {
+                        t = JavaTemplate.builder(sb.toString())
+                                .contextSensitive()
+                                .imports(String.format("org.junit.jupiter.api.Assertions.%s", assertMethod))
```

Comment:
I'm wondering if this should be 
```suggestion
                                .imports("org.junit.jupiter.api.Assertions")
```

Reply:
I used `AssertEqualsNullToAssertNull` as a reference for this recipe. It adds the imports this way to keep the existing style on the refactored file (e.g.,. if the refactored file is not using static imports, the class keep it that way.